### PR TITLE
Simplify: numerical-only adult

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Save a version of adult with only numerical columns

This dataset enables to simplify the first notebook.

It can also be reused in other places, to simplify exercises and examples (cc @glemaitre ) .

### CODE REVIEW

File: python_scripts/02_numerical_pipeline.py [hunk 1]
```diff
@@ -28,11 +28,16 @@
 #
 # We will use the same dataset "adult_census" described in the previous notebook.
 # For more details about the dataset see <http://www.openml.org/d/1590>.
+#
+# As Numerical data is the most natural type of data used in machine
```
Comment (glemaitre) [inline python_scripts/02_numerical_pipeline.py:32]:
```suggestion
# As numerical data is the most natural type of data used in machine
```

**Comment by GaelVaroquaux** [comment]
The created dataset has no NA. Looking at the list of available datasets ( https://github.com/INRIA/scikit-learn-mooc/issues/98 ) it seems to me that it fills a nice niche.

Unless there is strong outcry, I am planning to merge this tomorrow (not tonight, to give people time to look at decent hours), in the interest of quick iterations.

**Comment by glemaitre** [comment]
> Unless there is strong outcry

Not on my side. My personal preference would have been a middle-ground:

* identify the numerical with dtypes;
* select by column names using pandas.

My reason is that I would expect people to know `pandas` at this stage.

But I am fine with the proposal, it is just lowering the entry bar at the minimum.

**Comment by GaelVaroquaux** [comment]
> My reason is that I would expect people to know pandas at this stage.

Every line of code means a cognitive overload. The more a notebook or an exercise has lines of code, the harder it is for people to focus on the pedagogical point that it is making, in particular beginners.

The principle of reducing lines of code to focus on the message we are trying to pass through is one that we need to apply through the MOOC, unless we are doing an exercise / example for the purpose of teaching to fit everything together.

There is a lot of potential improvement along this line throughout the course.

**Comment by GaelVaroquaux** [comment]
OK, merging this.

**Pull request merged**